examples/lambda-update/step1/index.ts

```diff
@@ -31,7 +31,6 @@ export const graphqlPublicZip = new aws.s3.BucketObject(
   "lambda-source",
   {
     bucket: lambdaStore.id,
-    key: "samplefn",
```

Comment:
Why the change to this example?

Reply:
Both BucketObjects were created with the same key in the same bucket, so one overwrote the other. Removing the key means we'll just use auto-naming based on the Pulumi identifier. 

This was surfacing as a refresh diff sometimes during the test - where the version id of the object had changed. Essentially this is multiple data races: which one gets written first or second, then once written if one gets read before or after the other has also been written - and therefore if it actually gets a different version to what it puts 🤦

Reply:
Whoops that is bad. Thanks for clarifying!